beast/tests/test_regresscheck.py

```diff
@@ -129,33 +161,25 @@ def test_make_kurucz_tlusty_spectral_grid(self):
         # read in the cached isochrones
         oiso = ezIsoch(self.iso_fname_cache)
 
-        # define the distance
-        distances = [24.47]
-        distance_unit = units.mag
-
-        velocity = -300 * units.km / units.s
-        redshift = (velocity / const.c).decompose().value
-
-        # define the spectral libraries to use
-        osl = stellib.Tlusty() + stellib.Kurucz()
-
-        # define the extinction curve to use
-        extLaw = extinction.Gordon16_RvFALaw()
+        # remove the isochrone points with logL=-9.999
+        oiso.data = oiso[oiso["logL"] > -9]
```

Comment:
Will not be needed once PR #596 is merged as this cut is then done at a lower level (in `make_spectral_grid`).

Reply:
Perfect.  I'll remove it!

Reply:
Yeah!